### PR TITLE
1. Pushed my code to develop branch2. We can consider it as first fu…

### PR DESCRIPTION
**Pull request by chetan-kapadane**
…nctional commit.

### CODE REVIEW

File: src/Components/ToDo/index.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import Container from '@material-ui/core/Container';
+
+import store from '../../Components/ToDo/SagaStore/store';
+
+import AddToDo from './AddToDo';
+import ToDoList from './ToDoList';
+// import ResetButton from './resetButton';
+
+const ToDo = () => (
+  <Provider store={store}>
```
Comment (sunny-mody) [inline src/Components/ToDo/index.tsx:14]: use store Provider in src/index.tsx file

File: src/Components/ToDo/index.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <Container maxWidth="md">
+      <Typography style={{ padding: '20px', color: 'blue', margin: '20px' }} variant="h5">
```
Comment (sunny-mody) [inline src/Components/ToDo/index.tsx:16]: Don't use inline styles
try using makeStyles [https://material-ui.com/styles/advanced/#makestyles](https://material-ui.com/styles/advanced/#makestyles)

File: src/Components/ToDo/index.tsx (hunk 1 above)
Comment (chetan-kapadane) [reply src/Components/ToDo/index.tsx:14]: Sure, I'll update it to entry!

File: src/Components/ToDo/index.tsx (hunk 2 above)
Comment (chetan-kapadane) [reply src/Components/ToDo/index.tsx:16]: Correct, Some of issues I'm facing when trying to use makeStyles.
I'll update here.

**Pull request merged**